algorat/ratchelor
# Mobile update

Updates layouts to work on mobile screens

it does this by scaling down the main game as much as possible and moving some of the UI components over to a newly created panel on the right that uses the MobileWrapper component.

To test:
- visit ratchelor.web.app 
- play on both mobile and desktop and confirm behavior is as expected.
